app/views/katello/api/v2/packages/backend.json.rabl

```diff
@@ -5,3 +5,8 @@ attributes :children, :checksumtype, :size, :url, :build_time, :summary, :group,
 node :human_readable_size do |package|
   number_to_human_size(package.size) if package.size
 end
+
+node :human_readable_time do |package|
+   Time.at(package.build_time).strftime("%B %e, %Y at %I:%M %p") if package.build_time
```

Comment:
@dhananjay316 an extra space added in the front, it should only have 2 spaces. Rubocop doesn't like it 
~~~
[2019-11-18T21:41:37.191Z] Offenses:
[2019-11-18T21:41:37.191Z] 
[2019-11-18T21:41:37.191Z] /home/jenkins/workspace/katello-pr-test/app/views/katello/api/v2/packages/backend.json.rabl:10:1: C: Layout/IndentationWidth: Use 2 (not 3) spaces for indentation.
[2019-11-18T21:41:37.191Z]    Time.at(package.build_time).strftime("%B %e, %Y at %I:%M %p") if package.build_time
[2019-11-18T21:41:37.191Z] ^^^
[2019-11-18T21:41:37.191Z] /home/jenkins/workspace/katello-pr-test/app/views/katello/api/v2/packages/backend.json.rabl:12:1: C: Layout/TrailingBlankLines: 1 trailing blank lines detected.
~~~